docs/src/advanced-concepts/fine-tuning.rst

```diff
@@ -82,7 +68,8 @@ These parameters control whether to use LoRA for pre-trained model fine-tuning
 (``LORA_RANK``), and the regularization factor for the low-rank matrices
 (``LORA_ALPHA``).
 
-4. Run ``mtt train options.yaml`` to fine-tune the model.
+4. Run ``mtt train options.yaml -c best_model_*.ckpt`` to fine-tune the model.
```

Comment:
What does the * stand for here? Perhaps it would be worth explaining what it is

Reply:
I was referring to the fact that the checkpoint with best model is now forced to be have the `best_model_` prefix. But I think here I will just leave it as `best_model.ckpt`, since this is the default name now.

Reply:
> I was referring to the fact that the checkpoint with best model is now forced to be have the `best_model_` prefix

Where is this enforced? I don't see it in this PR

Reply:
> > I was referring to the fact that the checkpoint with best model is now forced to be have the `best_model_` prefix
> 
> Where is this enforced? I don't see it in this PR

It happens in the `Trainer.save_checkpoint()`, where I add `last_checkpoint_` and `best_` prefixes to the path

Reply:
Ok, this means that the final checkpoint for PET-MAD will be named `best_model_{something}.ckpt`, right?

Reply:
> It happens in the Trainer.save_checkpoint()

Ok, but then this is only something for PET, not a general thing. This should be fine here for the checkpoint name, but I now realize that this file (which should apply to any archiotecture)  contains a lot of PET-specific documentation (LoRA hypers, …). Should we move this elsewhere to be explicitly about PET?

Reply:
Ah, there is a warning on top. Might still make sense to move the file, but this can happen later

Reply:
> Does this mean that the published checkpoint for PET-MAD will be named `best_model_{something}.ckpt`? Or can it be renamed and still work?

Nono, for sure the final name of the file can be anything, it's just the default name (i.e. the one the checkpoint will have after the training) is forced to have this prefix to distinguish between the checkpoint with only the best model, and the one with the last step of the training. You can rename the model later and use it with `-c` normally.

Reply:
> Ah, there is a warning on top. Might still make sense to move the file, but this can happen later

Maybe we can do it while doing a PR on more general and high-level fine-tuning implementation, but for now I would keep it like this